ouroboros-consensus-test/ouroboros-consensus-test.cabal

```diff
@@ -326,3 +326,45 @@ test-suite test-infra
                        -Wredundant-constraints
                        -Wmissing-export-lists
                        -fno-ignore-asserts
+
+benchmark bench-mempool
```

Comment:
If we plan on writing more benchmarks for other consensus components, would we want to have a single benchmark executable or one for each component that we target?

Reply:
What about one per component? Why? To keep them isolated. But I don't know if the other approach as more advantages. Thoughts?

Reply:
I'm not opposed to having one per component. An advantage to one executable for all is that we can reuse code, but I don't know how often we will actually be in a situation where we can reuse code. I think one advantage of having multiple executables is that benchmark running times tend to be on the lengthy side, so not having one monolith executable that takes ages to run is probably a good thing.